apps/ui/src/views/Editor.vue

```diff
@@ -192,6 +193,12 @@ const extraContacts = computed(() => {
 
   return space.value.treasuries as Contact[];
 });
+const body_definition = computed(() => ({
```

Comment:
```suggestion
const bodyDefinition = computed(() => ({
```

Reply:
We could add an eslint for that.

Tried something like this inside ui/package.json eslintConfig:

```
"rules": {
    "@typescript-eslint/naming-convention": [
    "error",
    {
      "selector": ["variable"],
      "modifiers": ["const", "destructured"],
      "leadingUnderscore": "allow",
      "trailingUnderscore": "forbid",
      "format": ["snake_case", "UPPER_CASE", "camelCase"]
    },
    {
      "selector": "variable",
      "modifiers": ["const"],
      "leadingUnderscore": "allow",
      "trailingUnderscore": "forbid",
      "format": ["UPPER_CASE", "camelCase"]
    },
    {
      "selector": ["function"],
      "format": ["camelCase"],
      "leadingUnderscore": "allow",
      "trailingUnderscore": "forbid"
    }
```

Eberything's passing, there's just a single error in Editable.

Not sure about adding this rules inside the config package, as they're also used by other repos without any strict naming rules (else will require some refactoring on these repo)